package.json

```diff
@@ -12,10 +12,10 @@
   "author": "Grafana Labs <team@grafana.com> (https://grafana.com)",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@grafana/data": "latest",
-    "@grafana/runtime": "latest",
+    "@grafana/data": "7.3.0",
```

Comment:
7.4 is "latest" now... but we should still depend on 7.3 until we actually want to force it